transcendence-app/src/chat/infrastructure/db/postgres/chatmember.postgres.repository.ts

```diff
@@ -13,4 +18,31 @@ export class ChatMemberPostgresRepository
   constructor(protected pool: PostgresPool) {
     super(pool, table.CHATMEMBERS);
   }
+
+  async retrieveChatRoomMembers(
+    chatRoomId: string,
+  ): Promise<ChatMemberWithUser[] | null> {
+    const users = await makeQuery<ChatMemberWithUserEntity>(this.pool, {
+      text: `SELECT u."username",
+                    u."avatarId",
+                    u."avatarX",
+                    u."avatarY",
+                    c."ownerId",
```

Comment:
suggestion: What do you think about adding `c."ownerId" IS NOT NULL as owner,` to return the boolean from here?

Reply:
Will do. Good idea

Reply:
done